strings/en_US/messages.po

```diff
@@ -4505,18 +4505,12 @@ msgstr "Awaiting approval"
 msgid "Total awaiting approval"
 msgstr "Total awaiting approval"
 
-msgid "Thank you for your interest in joining the CoderDojo community!"
-msgstr "Thank you for your interest in joining the CoderDojo community!"
+msgid "Thank you for your interest in CoderDojo!"
+msgstr "Thank you for your interest in CoderDojo!"
 
```

Comment:
Sorry, I made a mistake in the line numbers I said to delete.  If you can put these ones back in?  (You should be able to 'Commit suggestion' below the comment.

```suggestion

msgid "The coronavirus pandemic continues to impact all our lives, including how Dojos can run."
msgstr "The coronavirus pandemic continues to impact all our lives, including how Dojos can run."
```